virtdata-lib-basics/src/test/java/io/nosqlbench/virtdata/library/basics/tests/long_string/CombinationsTest.java

```diff
@@ -69,11 +70,12 @@ public void testAZ09() {
         assertThat(combinations.apply(31)).isEqualTo("D1");
     }
 
-    @Test(expected = ArithmeticException.class)
+    @Test
     public void testOverflow() {
         // (104^9 / 2^63) < 1.0
         // (104^10 / 2^63) > 1.0
-        Combinations combinations = new Combinations(
+        assertThatExceptionOfType(ArithmeticException.class)
+                .isThrownBy(() -> new Combinations(
```

Comment:
it looks a bit suspicous that the exception comes from the constructor and not from the apply call. thoughts?